benches/crypto.rs

```diff
@@ -98,11 +100,26 @@ mod ed25519_benches {
             );
             c.bench_with_input(
                 BenchmarkId::new("BLS12381 batch verification", *size),
-                &(msg, blst_public_keys, blst_signatures),
+                &(
+                    msg.clone(),
+                    blst_public_keys.clone(),
+                    blst_signatures.clone(),
+                ),
                 |b, i| {
                     b.iter(|| VerifyingKey::verify_batch_empty_fail(&i.0, &i.1[..], &i.2[..]));
                 },
             );
+            c.bench_with_input(
+                BenchmarkId::new("BLS12381 aggregate verification", *size),
+                &(
+                    msg.clone(),
```

Comment:
do we need to clone on the last use in the function?

Reply:
No, its just left over from when it was done before the other benchmarks. I'll remove it.